main.py

```diff
@@ -296,24 +296,23 @@ def eval_worker(gpu, ngpu_per_node, args, res_queue=None):
              best_metric=best_metric, res_queue=res_queue, pad=args.pad)
 
 
-def writer(args, res_queue):
-    result_path = os.path.join(args.exp_dir, args.result_fname)
-    myprint("Writer process standing by... %s" % result_path, color='yellow')
+def writer(args, res_queue, files_path):
```

Comment:
another pedantic comment - instead of `files_path` (since it's only single file now), let's call it `input_file_path`?